dali/pipeline/operators/color/color_twist.h

```diff
@@ -187,6 +187,7 @@ class ColorTwistBase : public Operator<Backend> {
   const int C_;
 
   USE_OPERATOR_MEMBERS();
+  using Operator<Backend>::RunImpl;
```

Comment:
maybe make this part of `USE_OPERATOR_MEMBERS`?

Reply:
I was trying to do it, but some of the operators use it in a specializations for a certain backend (for example CPUBackend).
Than I would need to do `using Operator<CPUBackend>::RunImpl`. That would require either adding an argument to a macro, and specifying it everywhere, or having a variadic macro, which doesn't really work with no arguments unless you use a gcc extension.

And this macro only brings to scope stuff from OperatorBase which is not templated.

Reply:
fair enough